src/classes/GE_GiftEntryController.cls

```diff
@@ -147,7 +149,7 @@ public with sharing class GE_GiftEntryController {
             upsertDataImport(dataImport);
             return BGE_DataImportBatchEntry_CTRL.runSingleDryRun(dataImport.Id, batchId);
 
-        } catch (Exception ex) {
+        } catch (DmlException ex) {
```

Comment:
From our conversation with the team, I think we need to revert these changes and keep this as generic "Exception" for the time being

Reply:
done